### DISCUSSION TITLE
案例《看不见的罪行：我们是否让性侵受害者失望了？》

### DISCUSSION THREAD

**Discussion by meteoritesw** [data-journalism/data-journalism.github.io#35] [Show and tell]
### 数据新闻典型案例：看不见的罪行：我们是否让性侵受害者失望了？
链接：[https://www.smh.com.au/interactive/2019/are-we-failing-victims-of-sexual-violence-v-2/](url)
小组成员：常誉中、陈晨昱、王晓萱、薛敬文

### 一、简介
**1.	奖项背景**
2019年“信息之美奖”（The Kantar Information is Beautiful Awards）评选：《看不见的罪行：我们是否让性侵受害者失望了？》（The Invisible Crime: Are We Failing Victims Of Sexual Violence?）获得**人道主义类金奖**。

**2.	事件背景**
澳大利亚性暴力事件很普遍，却很少受到起诉。
澳大利亚统计局2016-17年的调查：18万澳大利亚人经历过性暴力，只有9%至14%的人向警方报案，更少的人胜诉。
案件通常在三个关键点中的一个被放弃——**向警方报案；提出指控；以及通过法庭程序。**
这篇报道试图用数据来揭示这一现象，通过数据分析为什么司法系统没有更好地处理性侵犯问题。

**3.	新闻内容**

**1. 第一部分：受害者**
   

>1. 有多少性暴力事件的受害者？
>2. 有多少受害者向警方报案？
>3. 有多少报告给警方的案件进入下一步指控环节？

   
**2. 第二部分：通往正义的漫长道路**

>1. 以新南威尔士州的数据为例，追踪性侵犯案件在澳大利亚这个人口最多的州的司法系统中的情况。
>2. 数据显示，案件通常在三个关键点中的一个被放弃——向警方报案；提出指控；以及通过法庭程序。

**3. 第三部分：性犯罪如何变得隐蔽（invisible）**

>1. 第一阶段：向警察报案
>案件没有进行的原因有很多，包括受害者感到太痛苦而无法继续，或者没有足够的证据。
>2. 第二阶段：立案（提出指控）
>司法系统层面：司法系统没有明确对性同意的定义，警务部门也没有绝对清晰的标准，检察部门为此损耗了人力。
>警官层面：警官人员可能存在配备不足的情况，他们的工作量也在不断增加。此外，警官往往对性暴力事件有一个默认的立场：这些案件无法进入下一步法律程序。
>3. 第三阶段：通过法庭程序
>即使警方确实提出指控，如果检察院认为定罪的机会不大，也可以中止案件的审理，且检察官不需要向法院解释为什么中止案件。
>交付听证会给了被告一个检验案件的机会——包括对受害者进行交叉询问。但这一阶段中对于犯罪细节的过分强调可能会助长受害者的创伤。
>长时间的拖延是性侵犯案件的一个特点。来自新南威尔士州的数据显示，对于性侵犯案件的定罪时间正稳步增加。

### 二、数据来源
①Australian Bureau of Statistics澳大利亚统计局（ABS Personal Safety Survey个人安全调查, 2016; ABS Recorded Crime Victims - Australia, 犯罪受害者调查2018）
②Victorian/Western Australia/Queensland crime statistics agency 维多利亚州/西澳大利亚州/昆士兰州犯罪统计局
③Victoria’s sexual offencers and criminal investigation team(SOCIT) 维多利亚州性犯罪和刑事调查小组
④NSW higher criminal courts新南威尔士州高等刑事法院
⑤BOCSAR新州犯罪统计和研究局

### 三、数据的选取与处理方法

> 报道将数据的选取与处理方法特别列出，成文发布在《澳大利亚时代报》，供读者参考。
> [https://www.theage.com.au/national/invisible-crime-about-the-data-20190807-p52eo8.html?_ga=2.258644362.1619314112.1631520437-52355398.1631342566](url)

此处主要介绍“对性暴力受害者人数的估计”以及“对性侵犯报案率的估计”这两项数据的选取与处理方法。

**1.	性暴力受害者的人数统计The estimate of the number of sexual violence victims**
解释内容：选取抽样调查的方法（ABS在2016.11-2017.6对21242人面对面访谈），对性侵犯、性威胁的定义，相对标准误差（11.6%，说明受害人数在18万至29万之间）。
选择个人安全调查而非犯罪受害者调查：更大的女性回答样本量，更低的相对标准误差，不是电话而是面对面访谈，受访者有更高的透露意愿
充分的解释与说明让读者在获得知情权的情况下对作者的处理更加信服。

**2.	对性侵犯的报案率的估计数 The estimate on the reporting rate for sexual assault**
计算方法：从澳大利亚统计局的犯罪受害者报案中获取报案的性侵犯受害者数量，并将其除以个人安全调查中性侵犯受害者人数的上下限。
结果：大约9%到14%的性侵犯被报告给警方。
“实际的受害者比向警方报告的受害者多7到11倍”。在说明计算方法的同时指出该数据与文中的联系，让读者更迅速地理解这组数据处理的意义。
> 影响数据处理准确性的不确定因素：
> ①两个调查开展的时间以及调查的时间范围并不相同
> ②两个调查对于性侵犯的定义略有不同
> ③数据未包含的一类人：童年时受到性侵犯，但向警方报告时已经成年的受害者
> ④犯罪受害者调查考虑了一个人受到多次性侵犯的情况，个人安全调查则没有
> 坦诚面对数据处理中不可避免的误差，更能体现出该篇报道的真实性，同时也能有效避免质疑。

### 四、数据分析

**1. 框架结构**
![1](https://user-images.githubusercontent.com/62281445/133062998-54ee6728-a2ac-45c5-a0b7-fe55a4737515.jpg)
**2. 叙述特点**
      1. 叙述目的明确。在引言部分，作者提出了性侵问题中最常见的一个问题：司法系统是否有更好的方法处理性侵？因此本报道全篇围绕此问题展开叙述，为读者介绍了性侵维权过程的曲折。
      2. 行文逻辑遵循案件的处理顺序。引言提到案件最容易撤销的三个阶段：报案、调查和法庭，下文便按此顺序介绍了各个环节中案件终止的受害者数量。
      3. 受害者经历的叙述比重较高。本报道着大量笔墨介绍受害者的经历，辅以真实的数据，让读者深刻体会到受害者的悲痛。

**3. 可视化处理**
①受害者散点图。
该报道用几幅散点图展示了受害者数目，每一个点代表一位受害者。
**这些散点图清晰地展示了众多性侵案件中只有少数人得以维权。**
<img width="660" alt="2" src="https://user-images.githubusercontent.com/62281445/133063073-14e3b6cd-6c19-400e-90aa-0970290c7dd2.png">
<img width="646" alt="3" src="https://user-images.githubusercontent.com/62281445/133063085-2ae60f25-1369-4297-95f0-2a355f1fd708.png">
<img width="637" alt="4" src="https://user-images.githubusercontent.com/62281445/133063102-0f76e7fe-0d29-4abc-9736-7fa534d4c8c8.png">
<img width="645" alt="5" src="https://user-images.githubusercontent.com/62281445/133063111-309d2ccc-6358-4335-ac73-1179e3e94f10.png">

②分流图
分流图展示了已报案的性侵犯案件的最终结果。
<img width="623" alt="6" src="https://user-images.githubusercontent.com/62281445/133063134-c5d18dd6-34bf-41c5-b0c3-2c7d2d4b99ac.png">

③条形图
条形图展示了性侵案件的处理时间逐年增加，从2010年到2018年，处理时间增加了45%。具体细分到两个部分，从逮捕到拘禁、从拘禁到结果的时间都逐年增加。
<img width="645" alt="7" src="https://user-images.githubusercontent.com/62281445/133063148-621e328d-94b9-4e6c-8a2b-154c8124012f.png">
<img width="648" alt="8" src="https://user-images.githubusercontent.com/62281445/133063163-d10bc62a-9a0a-43b2-a890-ffc85e0eabea.png">

**4. 可视化风格**
      1. 色调与情感基调：黑白两色为主。情感基调沉痛而冷静，悲愤而克制。
      2. 动态图表：文中每一张图片都是动态的，并随着读者的阅读进度不断变化。如：

>1. 在受害者散点图中，代表受害者的18000颗粒子在一个圆框中旋转，提醒着人们它们背后代表了一个个鲜活的生命
>2. 人物肖像由粒子合成，既起到了马赛克的作用，尊重受害者的隐私，又呼应了人物和性暴力受害者群体是点和面的关系
>3. 在分流图中，性犯罪事件的处理方向由堆积流动的颗粒显示比例大小，随着网页的下拉，颗粒一层层堆积向不同方向。

这个图表在可视化数据的同时也承担了叙事的效能，读者真切地感受到受害者维权困难重重，施暴者很少得到法律制裁

**5. 沉浸式阅读体验**
①动画效果：数据和图表随着读者的阅读进度而动态变化，阅读过程流畅，读者可以通过鼠标反复观看某个图表形成的过程。
②音画结合：嵌入了3段访谈受害者的音频，3位受害者分别讲述了自己受到的性暴力，警方的漠视和法庭作证带来的二次伤害。

### 五、意义
1.	兼顾理性搜证论证与感性人文关怀
2.	为公众理性讨论性暴力犯罪提供语境
3.	具有现实意义，问题解决导向

感谢大家的垂听和阅读！

**Comment by chengjun** [comment]
https://www.abs.gov.au/statistics/people/crime-and-justice/personal-safety-australia/2016#data-download
